### PR TITLE
test: expand unit tests 

### PR DESCRIPTION
**Pull request by yeul**
## Description

- Expands unit tests.

## Screenshots

N/A

## Testing instructions

N/A

### CODE REVIEW

File: Tests/PayPalMessagesTests/PayPalMessageModalTests.swift [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        let orientation = modalViewController.preferredInterfaceOrientationForPresentation
+
+        XCTAssertEqual(orientation, .portrait)
+    }
+
+    func testShouldAutorotate() {
+        let shouldAutoRotate = modalViewController.shouldAutorotate
+
+        XCTAssertFalse(shouldAutoRotate)
+    }
```
Comment (Seavenly) [inline Tests/PayPalMessagesTests/PayPalMessageModalTests.swift:78]: Would it reduce some noise if these were just combined into a single test that just tests initial property values?

**Review by Seavenly — COMMENTED** [review]
Are we able to update this value to 80% now to be our new pass threshold?
https://github.com/paypal/paypal-messages-ios/blob/3d095768746f75ea5079290e2d2210eb275cbab3/fastlane/Fastfile#L35

File: Tests/PayPalMessagesTests/PayPalMessageViewTests.swift [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        )
+
+        // Assert accessibility properties are correctly initialized
+        XCTAssertEqual(messageView.accessibilityTraits, .link)
+        XCTAssertTrue(messageView.isAccessibilityElement)
+        XCTAssertEqual(messageView.accessibilityLabel, Constants.accessibilityLabel)
+    }
+
+
+    func testStandardIntegrationInitialization() {
```
Comment (Seavenly) [inline Tests/PayPalMessagesTests/PayPalMessageViewTests.swift:55]: Would it make sense to move these last 2 tests into their own `PAyPalMessageConfigTests.swift` file?

File: Tests/PayPalMessagesTests/PayPalMessageModalTests.swift [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    
+    func testViewOnLoadingDelegate() {
+        let stateDelegateMock = PayPalMessageModalStateDelegateMock()
+        modalViewController.stateDelegate = stateDelegateMock
+
+        modalViewController.viewDidLoad()
+
+        XCTAssertTrue(stateDelegateMock.onLoadingCalled)
+    }
```
Comment (Seavenly) [inline Tests/PayPalMessagesTests/PayPalMessageModalTests.swift:57]: Is this test intended to be a duplicate of the one directly above?

Comment (yeul) [reply Tests/PayPalMessagesTests/PayPalMessageModalTests.swift:48]: Oops, yeah it's a dupe from when I was moving stuff around/renaming. Removed one of them.

File: Tests/PayPalMessagesTests/PayPalMessageViewTests.swift (hunk 2 above)
Comment (yeul) [reply Tests/PayPalMessagesTests/PayPalMessageViewTests.swift]: Moved those over to the ConfigTests file.

**Review by Seavenly — APPROVED** [review]

**Pull request merged**